CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
 
+### Notable Changes
+
+- The async_operation_image property of cumulus module should be updated to pull
```

Comment:
Is this a change in *this* PR?  Should we add it to the CHANGELOG somewhere else?     If it is, should it have this ticket as a header? @jennyhliu

Reply:
No. Just move the section up